src/main/resources/db/db.changelog-setup.xml

```diff
@@ -0,0 +1,267 @@
+            </column>
+            <column name="question_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="feedback_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="rate" type="INT">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet id="1.0.9" author="volodymyr.protsiv">
+        <createTable tableName="user_roles">
+            <column name="role_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="user_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet id="1.0.10" author="volodymyr.protsiv">
+        <createTable tableName="course_group">
+            <column name="group_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="course_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+    <changeSet id="1.0.11" author="volodymyr.protsiv">
+        <createTable tableName="course_feedback_request_question">
+            <column name="feedback_request_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+            <column name="question_id" type="BIGINT">
+                <constraints nullable="false"/>
+            </column>
+        </createTable>
+    </changeSet>
+
+    <changeSet id="1.0.12" author="volodymyr.protsiv">
+        <addPrimaryKey
+                columnNames="user_id, role_id"
+                constraintName="user_roles_pkey"
+                schemaName="public"
+                tableName="user_roles"
+        />
+    </changeSet>
+
+    <changeSet id="1.0.13" author="volodymyr.protsiv">
+        <addPrimaryKey
+                columnNames="group_id, course_id"
+                constraintName="course_group_pkey"
+                schemaName="public"
+                tableName="course_group"
+        />
+    </changeSet>
+
+    <changeSet id="1.0.14" author="volodymyr.protsiv">
+        <addPrimaryKey
+                columnNames="feedback_request_id, question_id"
+                constraintName="course_feedback_request_questions_pkey"
+                schemaName="public"
+                tableName="course_feedback_request_question"
+        />
+    </changeSet>
+
+    <changeSet id="1.0.15" author="volodymyr.protsiv">
+        <addForeignKeyConstraint baseColumnNames="role_id" baseTableName="user_roles" constraintName="user_roles_roles_fkey" deferrable="false" initiallyDeferred="false" onDelete="NO ACTION" onUpdate="NO ACTION" referencedColumnNames="id" referencedTableName="role"/>
```

Comment:
Please, add these constraints directly in changeset where you create tables

Reply:
Added constraints directly. 
Added db.changelog-insert-role-permission.xml with role and permission insertions. @Forfend